samples/subsys/usb/mass/src/main.c

```diff
@@ -113,6 +117,8 @@ static int mount_app_fs(struct fs_mount_t *mnt)
 		mnt->mnt_point = "/RAM:";
 	} else if (IS_ENABLED(CONFIG_DISK_DRIVER_SDMMC)) {
 		mnt->mnt_point = "/SD:";
+	} else if (IS_ENABLED(CONFIG_DISK_DRIVER_MMC)) {
+		mnt->mnt_point = "/SD:";
```

Comment:
`CONFIG_MMC_VOLUME_NAME` is used to create the disk name in the disk driver subsystem.
It defaults to "SD" when `CONFIG_FAT_FILESYSTEM_ELM` is enabled, which is the guard over this code block.
Dynamically constructing this name based on `CONFIG_MMC_VOLUME_NAME` would be pointless because the FAT zephyr shim only works with hardcoded disk names anyway.
The whole disk name thing is a giant mess. I had a first go at improving things in #75841 but bit off too much at once.

Reply:
Flash disk can be used without filesystem and therefore the LUN defined above won't work. The first parameter in `USBD_DEFINE_MSC_LUN` is used as disk name in disk driver interface. Therefore, I believe it should be set to `CONFIG_MMC_VOLUME_NAME`. I don't quite remember the reason behind the use of `STRINGIFY` in `USBD_DEFINE_MSC_LUN` which may stand in line here...